M2/Macaulay2/m2/set.m2

```diff
@@ -15,7 +15,7 @@ tally String      :=
 tally VisibleList := Tally => tally
 
 elements = method()
-elements Tally := x -> splice apply(pairs x, (k,v) -> v:k)
+elements Tally := toList Tally := x -> splice apply(pairs x, (k,v) -> v:k)
```

Comment:
Could you document  `(toList, Tally)` (perhaps just adding it to the documentation of `(elements, Tally)`?

Reply:
decided to add it to `toList`... but why does the doc say that the input can be an integer??
```
A, a set, a string, a basic list, *or an integer*, or an instance of a class with the iterator method installed
```